### PR TITLE
feat(config): add/correct config files for iSurpass J1825

### LINKED ISSUE #4839: Missing device configuration: iSurpass J825
### Checklist

- [X] It is not in the [configuration DB](https://devices.zwave-js.io/?search=3)

- [X] It was not [merged recently](https://github.com/zwave-js/node-zwave-js/pulls?q=is%3Apr+is%3Aclosed) or has a [pending PR](https://github.com/zwave-js/node-zwave-js/pulls?q=is%3Aopen+is%3Apr)

### Which device is missing?

iSurpass J825

### Manufacturer ID

0x021c

### Product Type

0x5010

### Product ID

0x1001

### Firmware Version

2.16

### Is the device listed on the Z-Wave Alliance website?

No

### Do you have a manual?

[J1825 Wall Reader Instruction.pdf](https://github.com/zwave-js/node-zwave-js/files/9175606/J1825.Wall.Reader.Instruction.pdf)

### PR DESCRIPTION
**Pull request by cyrilst**
Config file for iSurpass J1825 imported from OpenHAB

fixes: https://github.com/zwave-js/node-zwave-js/issues/4839

### CODE REVIEW

**Comment by cyrilst** [comment]
A small explanation:
- productType 0x5010 / productId 0x1001 is the "Wall Reader" (keyboard without door lock and door handle).
- iLock and iLock15 are the same product (door lock with door handle)
- I'm pretty sure dl9101v is sometimes called iLock…

File: packages/config/config/devices/0x021c/ilock15.json [hunk 1]
```diff
@@ -5,8 +5,8 @@
 	"description": "Smart Door Lock",
 	"devices": [
 		{
-			"productType": "0x5010",
-			"productId": "0x1001",
+			"productType": "0x634b",
+			"productId": "0x504c",
 			"zwaveAllianceId": 3048
```
Comment (AlCalzone) [inline packages/config/config/devices/0x021c/ilock15.json]: https://products.z-wavealliance.org/products/3048 lists the product ID you moved to the new file, along with a picture of a lock. Not sure what's correct, but this doesn't seem to be long here (anymore).

File: packages/config/config/devices/0x021c/j1825.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+	"manufacturer": "Shenzhen iSurpass Technology Co. , Ltd",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x021c/j1825.json]:
```suggestion
	"manufacturer": "Shenzhen iSurpass Technology Co., Ltd",
```

Comment (AlCalzone) [reply packages/config/config/devices/0x021c/j1825.json]: please apply to the other file aswell

**Review by AlCalzone — CHANGES_REQUESTED** [review]

File: packages/config/config/devices/0x021c/j1825.json (hunk 2 above)
Comment (cyrilst) [reply packages/config/config/devices/0x021c/j1825.json]: Done. Thanks for the review.

File: packages/config/config/devices/0x021c/ilock15.json (hunk 1 above)
Comment (cyrilst) [reply packages/config/config/devices/0x021c/ilock15.json]: Also done :)

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**